### PR TITLE
This commit implements db*() methods from DBI and adds a new dbCreateTableAs() method

### PR DESCRIPTION
**Pull request by jarodmeng**
This commit implements db*() methods from DBI and adds a new dbCreateTableAs() method for PrestoConnection:
* Implement dbGetRowsAffected() for dbExecute() to work
* Implement dbRemoveTable()
* Implement dbCreateTable()
* Add a dbCreateTableAs() method for PrestoConnection
* Update NAMESPACE and documentations
* Update and add unit tests

### CODE REVIEW

**Comment by jarodmeng** [comment]
Contribute to fixing #156

**Comment by jarodmeng** [comment]
All unit tests passed and R CMD CHECK returns no warnings/errors/notes.

File: tests/testthat/test-dbCreateTable.R [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ),
+    'CREATE TEMPORARY TABLE is not supported in Presto'
+  )
+  expect_true(
+    dbCreateTable(
+      conn, test_table_name,
+      fields = c("city_id"="BIGINT", "city_name"="VARCHAR")
+    )
+  )
+  expect_true(dbExistsTable(conn, test_table_name))
```
Comment (leepface) [inline tests/testthat/test-dbCreateTable.R:63]: Can we add a test that column names and data types are correct?

**Review by leepface — CHANGES_REQUESTED** [review]

File: tests/testthat/test-dbCreateTable.R (hunk 1 above)
Comment (jarodmeng) [reply tests/testthat/test-dbCreateTable.R:63]: Good suggestion. Done. PTAL.

**Review by jarodmeng — COMMENTED** [review]
PTAL

**Pull request merged**